fastchat/conversation.py

```diff
@@ -1190,6 +1190,32 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
+# Yi template
+# reference: https://huggingface.co/01-ai/Yi-34B
+register_conv_template(
+    Conversation(
+        name = "yi-34b",
+        system_template="<|system|>\n{system_message}",
+        roles=("<|user|>", "<|assistant|>"),
+        sep_style=SeparatorStyle.ADD_COLON_TWO,
+        sep="\n",
+        sep2="\n\n",
+    )
+)
+
+# Yi template
+# reference: https://huggingface.co/01-ai/Yi-6B
+register_conv_template(
+    Conversation(
+        name = "yi-6b",
+        system_template="<|system|>\n{system_message}",
+        roles=("<|user|>", "<|assistant|>"),
+        sep_style=SeparatorStyle.ADD_COLON_TWO,
+        sep="\n",
+        sep2="\n\n",
+    )
+)
+
```

Comment:
I don't think you need to separate them. we only need one chat template for the two.

Reply:
can you refer to llama for example?